xcode/Subconscious/Shared/AppTheme.swift

```diff
@@ -122,6 +122,21 @@ extension Color {
     static let brandBgBlack = Color(red: 35/255, green: 31/255, blue: 32/255, opacity: 1) // #231F20
     static let brandBgSlate = Color(red: 57/255, green: 50/255, blue: 84/255, opacity: 1) // #393254
     
+    static let bgGradientLight = LinearGradient(
```

Comment:
Non-blocking: is this general enough that it ought to be extracted to theme? Or should it be view-specific?

Reply:
I think I'll move it to something more specific like `OnboardingTheme`.